tests/lib/rules/no-unused-prop-types.js

```diff
@@ -1387,15 +1319,16 @@ ruleTester.run('no-unused-prop-types', rule, {
       code: [
         'const PagingBlock = function(props) {',
         '  return (',
-        '    <SomeChild {...props} />',
+        '    <span>{...props}</span>',
```

Comment:
Oh I see, good point. Let me update the test so `SomeChild` is a thing then. I will look into creating the issue with babel-eslint.